.github/workflows/test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           otp-version: 22.2
           elixir-version: 1.10.4
+          experimental-otp: true
```

Comment:
Is there a reason for this? I thought OTP 22.2 is a stable version and can be used on the latest ubuntu image without issue 🤔

Reply:
Oh nevermind, I saw it was causing an SSL error on the CI 😅